doc/languages-frameworks/javascript.section.md

```diff
@@ -217,6 +222,15 @@ $ prefetch-npm-deps package-lock.json
 sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
 ```
 
+#### fetchNpmDeps {#javascript-buildNpmPackage-fetchNpmDeps}
+
+`fetchNpmDeps` is a Nix function that requires the following mandatory arguments:
+
+- `src`: A directory / tarball with `package-lock.json` file
+- `hash`: The output hash of the node dependencies defined in `package-lock.json`.
+
+And it returns a derivation with all `package-lock.json` dependencies downloaded into `$out/`, useable as a NodeJS cache, when you need to access it manually.
```

Comment:
```suggestion
It returns a derivation with all `package-lock.json` dependencies downloaded into `$out/`, usable as an npm cache.
```

Tiny voice nit (feel free to ignore that one, I don't feel that strongly) and clarifying it's an npm cache, not a generalized Node.js cache